### PR TITLE
Test Entity > Child record loop > Sort records TC_025.05

### PR DESCRIPTION
**Pull request by EvgeniyaPiskunova**
https://trello.com/c/MpIp6x9J/578-tc02505-entity-child-record-loop-sort-records
https://trello.com/c/tDGXkNsR/362-us02505-entity-child-record-loop-sort-records
https://trello.com/c/q9yZpySu/635-at0000000-entity-entity-title-action

### CODE REVIEW

File: src/test/java/EntityChildRecordsLoopTest.java [hunk 1]
```diff
@@ -137,33 +140,69 @@ public void testEditChildRecordsLoopDeleteRow() {
...
-     *       Start balance: -22.30
-     *       End balance: 18.30
+    /**
+     * * Precondition: **
+     * For the logged in user, there is one entry in the Child Records Loop, with the following attributes:
+     * Start balance: -22.30
+     * End balance: 18.30
      * ** Steps to reproduce: **
-     *       1. In the left menu, find and select the "Child Records Loop" item.
+     * 1. In the left menu, find and select the "Child Records Loop" item.
```
Comment (AnastasiiaPV) [inline src/test/java/EntityChildRecordsLoopTest.java:149]: Should delete all the coments

Comment (EvgeniyaPiskunova) [reply src/test/java/EntityChildRecordsLoopTest.java]: reverted all changes in other people code.

**Review by ArinaJur — APPROVED** [review]
Good Job!

**Review by AnastasiiaPV — APPROVED** [review]

File: src/test/java/EntityChildRecordsLoopTest.java (hunk 1 above)
Comment (AnastasiiaPV) [reply src/test/java/EntityChildRecordsLoopTest.java]: What do you mean?

**Pull request merged**